src/View/Widgets/AppMenu.vala

```diff
@@ -83,10 +83,14 @@ public class Files.AppMenu : Gtk.Popover {
             action_name = "win.singleclick-select"
         };
 
-        var foldes_before_files = new Granite.SwitchModelButton (_("Sort Folders before Files")) {
+        var folders_before_files = new Granite.SwitchModelButton (_("Sort Folders before Files")) {
             action_name = "win.folders-before-files"
         };
 
+        var restore_tabs = new Granite.SwitchModelButton (_("Restore Tabs on Starting")) {
```

Comment:
Not sure if this wording is best? When I see "start-up" I think more the machine/OS starting up. I am almost tempted to keep it layman and suggest "Restore Tabs from Last Time" but I am not totally convinced that is a good alternative lol

Reply:
How about "Restore Tabs when Launching"

Reply:
I feel like "when Launching" might be too ambiguous. 

Web calls this "Restore Tabs on Startup" which would we could go with for consistency
Chrome calls it "Continue where you left off" which feels super ambiguous :grimacing:
Firefox calls this "Show Tabs from Last Time" which I think validates @zeebok here and might be the most clear

Reply:
OK, I'll go with @zeebok 's suggestion then.